scripts/covid_tracking_project/historic_state_data/config.txt

```diff
@@ -0,0 +1,14 @@
+import: {
+    import_name: "COVIDTracking_States"
+    provenance_url: "https://www.covidtracking.com"
+    provenance_description: "COVID-19 related data in US states level."
```

Comment:
Mention that it's from The COVID Tracking Project?

Reply:
Ya!